mozilla-release/toolkit/mozapps/installer/package-name.mk

```diff
@@ -12,7 +12,7 @@ ifndef PACKAGE_NAME_MK_INCLUDED
 PACKAGE_NAME_MK_INCLUDED := 1
 
 ifndef MOZ_PKG_VERSION
-MOZ_PKG_VERSION = $(MOZ_APP_VERSION)
+MOZ_PKG_VERSION = $(MOZ_APP_VERSION_DISPLAY)
```

Comment:
Also need to add changes everywhere in scripts, where we use browser/config/version.txt file. Must be replaced accurately with browser/config/version_display.txt